src/LinuxTracing/LinuxTracingUtils.cpp

```diff
@@ -104,29 +105,26 @@ static ErrorMessageOr<std::string> ReadCgroupContent(pid_t pid) {
 }
 
 // Extract the cpuset entry from the content of /proc/<pid>/cgroup.
-std::optional<std::string> ExtractCpusetFromCgroup(const std::string& cgroup_content) {
-  std::istringstream cgroup_content_ss{cgroup_content};
-  std::string cgroup_line;
-  while (std::getline(cgroup_content_ss, cgroup_line)) {
-    if (cgroup_line.find("cpuset:") != std::string::npos ||
-        cgroup_line.find("cpuset,") != std::string::npos) {
+std::optional<std::string> ExtractCpusetFromCgroup(std::string_view cgroup_content) {
+  for (std::string_view cgroup_line : absl::StrSplit(cgroup_content, '\n')) {
+    if (absl::StrContains(cgroup_line, "cpuset:") || absl::StrContains(cgroup_line, "cpuset,")) {
```

Comment:
This is the only change that is more than mechanical. I had to replace the usage of stringstreams which can't operate on string_views.